rotkehlchen/tests/api/test_uniswap.py

```diff
@@ -1192,3 +1195,93 @@ def test_get_uniswap_trades_history_v3(rotkehlchen_api_server):
     assert len(trades) == 1
     assert '0x000e62ea8bf5f69e1e32920600839f56313c490f' in trades.keys()
     assert trades['0x000e62ea8bf5f69e1e32920600839f56313c490f'] == expected_trades
+
+
+@pytest.mark.parametrize('ethereum_accounts', [[LP_V3_HOLDER_ADDRESS]])
+@pytest.mark.parametrize('ethereum_modules', [['uniswap']])
+@pytest.mark.parametrize(
+    'start_with_valid_premium,ethrpc_endpoint,ethereum_manager_connect_at_start',
+    SKIPPED_UNISWAP_TEST_OPTIONS,
+)
+def test_get_v3_balances(rotkehlchen_api_server):
+    """Check querying the uniswap balances v3 endpoint works. Mock real data"""
+    mock_response = {
+        "result": {
+            "0xCC6A4c77D94957feADBD27DBdB4CDD3Ed00fc1E6": [
+                {
+                    "address": "0x8ad599c3A0ff1De082011EFDDc58f1908eb6e6D8",
+                    "nft_id": 226583,
+                    "price_range": "2659.805689681604<>3203.499390234706",
+                    "assets": [
+                        {
+                            "asset": "_ceth_0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+                            "total_amount": None,
```

Comment:
Again, why is this amount `None`? You have defined it constant `None` and I believe you don't update it later